<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="1" failed="0" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="197" started-at="2021-11-01T14:15:15 IST" finished-at="2021-11-01T14:15:15 IST">
    <groups>
    </groups>
    <test name="Default test" duration-ms="197" started-at="2021-11-01T14:15:15 IST" finished-at="2021-11-01T14:15:15 IST">
      <class name="Test.TestGetPerson">
        <test-method status="FAIL" signature="init()[pri:0, instance:Test.TestGetPerson@72a7c7e0]" name="init" is-config="true" duration-ms="17" started-at="2021-11-01T14:15:15 IST" finished-at="2021-11-01T14:15:15 IST">
          <exception class="java.lang.ExceptionInInitializerError">
            <full-stacktrace>
              <![CDATA[java.lang.ExceptionInInitializerError
at Test.TestGetPerson.init(TestGetPerson.java:22)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:176)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:122)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.lang.NullPointerException
at java.util.Properties$LineReader.readLine(Unknown Source)
at java.util.Properties.load0(Unknown Source)
at java.util.Properties.load(Unknown Source)
at com.scrolltest.api.utils.ConfigManager.<init>(ConfigManager.java:15)
at com.scrolltest.api.utils.ConfigManager.getInstance(ConfigManager.java:26)
at com.scrolltest.api.helpers.PersonServiceHelper.<clinit>(PersonServiceHelper.java:22)
... 27 more
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.ExceptionInInitializerError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- init -->
        <test-method status="SKIP" signature="TestGetAllPerson()[pri:0, instance:Test.TestGetPerson@72a7c7e0]" name="TestGetAllPerson" duration-ms="0" started-at="2021-11-01T14:15:15 IST" finished-at="2021-11-01T14:15:15 IST">
          <exception class="java.lang.ExceptionInInitializerError">
            <full-stacktrace>
              <![CDATA[java.lang.ExceptionInInitializerError
at Test.TestGetPerson.init(TestGetPerson.java:22)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:176)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:122)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.lang.NullPointerException
at java.util.Properties$LineReader.readLine(Unknown Source)
at java.util.Properties.load0(Unknown Source)
at java.util.Properties.load(Unknown Source)
at com.scrolltest.api.utils.ConfigManager.<init>(ConfigManager.java:15)
at com.scrolltest.api.utils.ConfigManager.getInstance(ConfigManager.java:26)
at com.scrolltest.api.helpers.PersonServiceHelper.<clinit>(PersonServiceHelper.java:22)
... 27 more
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.ExceptionInInitializerError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TestGetAllPerson -->
      </class> <!-- Test.TestGetPerson -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
